templates/core/features/index.html

```diff
@@ -4,314 +4,276 @@
-            <button class="p-tabs__link" role="tab" aria-selected="true" aria-controls="snaps-tab" id="snaps-tab-cta">Snaps</button>
-          </div>
-          <div class="p-tabs__item">
-            <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="snapd-tab" id="snapd-tab-cta" tabindex="-1">Snapd</button>
-          </div>
-          <div class="p-tabs__item">
-            <button class="p-tabs__link" role="tab" aria-selected="false" aria-controls="snapcraft-tab" id="snapcraft-tab-cta" tabindex="-1">Snapcraft</button>
-          </div>
+  <div class="row--25-75">
+    <div class="col">
+      <hr />
+      <div class="row">
+        <div class="col-3 col-medium-2 col-start-medium-2">
+          <ul class="js-tabbed-content p-tabs--vertical is-black u-hide--small" role="tablist"
+            aria-label="Dev to deployment">
+            <li>
+              <button class="p-tabs__item" role="tab" aria-selected="true" aria-controls="snaps-tab" id="snaps-tab-cta">Snaps</button>
+            </li>
+            <li>
+              <button class="p-tabs__item" role="tab" aria-selected="false" aria-controls="snapd-tab" id="snapd-tab-cta" tabindex="-1">Snapd</button>
+            </li>
+            <li>
+              <button class="p-tabs__item" role="tab" aria-selected="false" aria-controls="snapcraft-tab" id="snapcraft-tab-cta" tabindex="-1">Snapcraft</button>
+            </li>
+            <li>
+              <button class="p-tabs__item" role="tab" aria-selected="false" aria-controls="ubuntu-tab" id="ubuntu-tab-cta" tabindex="-1">Ubuntu-image</button>
+            </li>
+          </ul>
+          <form class="u-hide--large u-hide--medium">
+            <select name="boardSelect" id="boardSelect">
+              <option value="snaps-tab" selected="">Snaps</option>
+              <option value="snapd-tab">Snapd</option>
+              <option value="snapcraft-tab">Snapcraft</option>
+              <option value="ubuntu-tab">Snapcraft</option>
+            </select>
+          </form>
         </div>
-
-        <div tabindex="0" role="tabpanel" id="snaps-tab" aria-labelledby="snaps-tab-cta">
-          <div class="row u-equal-height">
-            <div class="col-8">
-              <h3>Snaps</h3>
-              <p>Ubuntu Core is built from snaps, a secure, confined, dependency-free, cross-platform Linux packaging format. Snaps are entirely self-contained, even to the point of encapsulating their own file system. This means they include everything they need to run in any environment. They're used by Ubuntu Core to both compose the image that's run on a device, and to deliver consistent and reliable software updates, even to low-powered, inaccessible and remotely administered embedded and IoT systems.</p>
+        <div class="col-6 col-medium-3">
+          <div tabindex="0" role="tabpanel" id="snaps-tab" aria-labelledby="snaps-tab-cta">
+            <div class="row u-equal-height">
+              <h5>Snaps</h5>
```

Comment:
I don't think this is needed, the tabs already show the title and it isn't on the design